src/tasks/LibraryBuilder/Templates/autoinit.c

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+
+#include <mono/jit/jit.h>
+#include <mono/jit/mono-private-unstable.h>
+#include <mono/metadata/assembly.h>
+
+#include "shared_library_log.h"
+
+static const char *bundle_path;
+
+void register_aot_modules (void);
```

Comment:
Since you now have a header, I think you should move the prototypes into that header instead.

Reply:
I originally was thinking that if users had their own runtime callback, then the prototypes would be unneeded (yet they would still be linked in via the header file). But on second thought, they would still need to invoke these right? Or are there ways to initialize the runtime without `register_aot_modules` with AOT on?